### DISCUSSION TITLE
How to create an empty record with no data

### DISCUSSION THREAD

**Discussion by Vishwaas** [prisma/prisma#25202] [Q&A]
### Question

I am using postgres with Prisma in my nodejs project.

I have a prisma model with the below schema
```prisma
model ParentObject {
  parent_object_iid        String           @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
  ChildObjects ChildObject[]
  SiblingObjects SiblingObject[]
} 
```
When I execute
```js
parentObject = await prisma.parentObject.create({ data: {} });
```
or 
```js
parentObject = await prisma.parentObject.create({});
```

Is there a different method I need to be using like `await prisma.parentObject.createRecordWithNoData()` or is this a bug?

### Expected behavior (optional)

I expect an empty record created with a unique random UUID generated and returned. But this does not create any new record but returns the first record in the table.

### Information about Prisma Schema, Client Queries and Environment (optional)

It works if the schema is
```prisma
model ParentObject {
  parent_object_iid        String           @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
  ChildObjects ChildObject[]
  SiblingObjects SiblingObject[]
  name string
}
``` 
and executing

```js
parentObject = await prisma.parentObject.create({ data: {name: 'john doe'} });
```

**Comment by ludralph** [comment]
Hi @Vishwaas 👋 

Thank you for raising this question.

I can see that you have defined a model called `ParentObject` in your schema. 

When executing your query, you're using `parentAccount`. Did you mean to write `parentObject`?
i.e:
```js
const parentAccount = await prisma.parentObject.create();
```
or 

```js
const parentAccount = await prisma.parentObject.create({ data: {} });
```

**Reply by Vishwaas** [reply]
yes, updated thank you

**Reply by ludralph** [reply]
Hi @Vishwaas 

Can you please confirm if this issue is resolved for you?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
